revolv/payments/tests/models/test_payment.py

```diff
@@ -418,3 +419,89 @@ def test_admin_reinvestment(self):
         user2 = RevolvUserProfile.objects.get(pk=user2.pk)
         self.assertEquals(user1.reinvest_pool, 50.00)
         self.assertEquals(user2.reinvest_pool, 150.00)
+
+    def test_admin_reinvestment_category_preference(self):
+        """
+        Test reinvestment on AdminReinvestment level. Lots of moving parts,
+        see docstrings for respective models for info.
+
+        Creates 3 users, 2 of which have category preferences that match
+        categories for project2. Checks that those 2 users will reinvest into
+        it completely, and that the other users do not.
+        """
+        user1, user2, user3 = RevolvUserProfile.factories.base.create_batch(3)
+        admin1 = RevolvUserProfile.factories.admin.create()
+        project1, project2 = Project.factories.base.create_batch(2)
```

Comment:
I think it would be good for the purposes of this test to define funding_goals for these projects